### ISSUE TITLE
[Flaky Test] Supplemental Claim Edit

### ISSUE THREAD

**Issue by pkarman** [department-of-veterans-affairs/caseflow#13794]
## Description

```
Supplemental Claim Edit issues when there is a rating end product when a user can withdraw issues show withdrawn issue when edit page is reloaded
     Failure/Error: expect(page).to have_content("Requested issues\n1. Left knee granted")
       expected to find text "Requested issues\n1. Left knee granted" in "CaseflowIntake\n  |   Smith, Bob (100000017)\nDSUSER (DSUSER)\nEdit Issues\nEstablished Thu May 30 2019 at 00:00 by DSUSER\nForm Decision Review Request: Supplemental Claim — VA Form 20-0995\nVeteran Bob Smith (100000017)\nReceipt date of this form 05/10/2019\nBenefit type Compensation\nClaimant Bob Vance, Spouse (payee code 10)\nWhen you finish making changes, click \"Save\" to continue.\nRequested issues\n2. Left knee granted\nDecision date: 05/10/2019\nSelect action\nWithdraw issue\nRemove issue\nWithdrawn issues\n1. PTSD denied\nDecision date: 05/10/2019\nWithdrawn on 05/28/2019\n+ Add issue\nCancelSave\n2 issues\nBuilt with ♡ by the Digital Service at VA\nTrack Caseflow Status|Send feedback"
     # ./spec/feature/intake/supplemental_claim/edit_spec.rb:662:in `block (4 levels) in <top (required)>'
     # ./spec/spec_helper.rb:23:in `block (2 levels) in <top (required)>'
```

## Background/context/resources

 - Circle CI Error:  [SC](https://app.circleci.com/pipelines/github/department-of-veterans-affairs/caseflow/29036/workflows/8c58681c-4145-42d9-8bc6-ee7d5f8e9550/jobs/107043)
 - Has the test already been skipped in the code?
   - [ ] Skipped
   - [x] Not Skipped
 - Related Flakes
    + 

## Approach

Time box this investigation and fix.

Remember that if a test has been skipped for a decent amount of time, it may no longer map to the exact code.

If you reach the end of your time box and don't feel like the solution is in sight:
  - [ ] document the work you've done, including dead ends and research
  - [ ] skip the test in the code
  - [ ] file a follow on ticket
  - [ ] close this issue

**Comment by pkarman** [comment]
possible fix in edd0e48

**Comment by leikkisa** [comment]
Closing due to fix in:
https://github.com/department-of-veterans-affairs/caseflow/pull/13792
